api/src/modules/indicators/indicator.repository.ts

```diff
@@ -2,4 +2,11 @@ import { EntityRepository, Repository } from 'typeorm';
 import { Indicator } from 'modules/indicators/indicator.entity';
 
 @EntityRepository(Indicator)
-export class IndicatorRepository extends Repository<Indicator> {}
+export class IndicatorRepository extends Repository<Indicator> {
+  //TODO add response DTO
```

Comment:
this seems to be unrelated. Also, you dont need a response DTO here

Reply:
Yep, this went here by mistake while I was implementing the indicator-data feature. THX for pointing out